src/Raven.Client/Util/CertificateLoaderUtil.cs

```diff
@@ -85,10 +85,12 @@ private static X509KeyStorageFlags AddUserKeySet(X509KeyStorageFlags? flag)
     [Conditional("DEBUG")]
     private static void DebugAssertDoesntContainKeySet(X509KeyStorageFlags? flags)
     {
-        const X509KeyStorageFlags keyStorageFlags = 
-            X509KeyStorageFlags.UserKeySet | 
-            X509KeyStorageFlags.MachineKeySet | 
-            (X509KeyStorageFlags)0x20 ; // X509KeyStorageFlags.EphemeralKeySet not defined in .Netstandard
+        const X509KeyStorageFlags keyStorageFlags =
+#if NETCOREAPP_3_1_OR_GREATER
```

Comment:
it looks like the key is NETCOREAPP3_1_OR_GREATER (VS should give you intellisense on that)

Reply:
done